third_party/move/move-compiler-v2/src/pipeline/ability_processor.rs

```diff
@@ -172,6 +172,7 @@ impl<'a> TransferFunctions for CopyDropAnalysis<'a> {
         let temp_needs_drop = |temp: &TempIndex| {
             self.target.get_local_type(*temp).is_reference() || exit_state.may_return()
         };
+        let live_srcs_in_dests = live_var.live_src_temps_in_dests(instr);
```

Comment:
Only need this for few instructions so don't need to eagerly compute this all the time. But I would do this differently anyway.

Reply:
done